Tests/SwerverTests/ResponseSpec/HttpResponseSpec.swift

```diff
@@ -5,48 +5,104 @@ import Foundation
 
 class HttpResponseSpec: QuickSpec {
     override func spec() {
-        describe("HttpResponse") {
-            var responseBuilder = ResponseBuilder()
+        describe("A Reponse object") {
+            func summonBuilder() -> HttpResponseBuilder {
+                return HttpResponseBuilder()
+            }
             
-            it("can equate two response object") {
-                let responseOne = responseBuilder
-                    .with(statusCode: .ok)
-                    .with(contentType: .text)
-                    .with(body: String())
-                    .build()
-                
-                let responseTwo = responseBuilder
-                    .with(statusCode: .ok)
-                    .with(contentType: .text)
-                    .with(body: String())
-                    .build()
-                
-                expect(responseOne).to(equal(responseTwo))
+            context("when compares with another response object") {
+                it("is true when attributes match") {
+                    var responseBuilder = summonBuilder()
+                    
+                    let responseOne = TestData.buildResponse()
```

Comment:
I like the idea of abstracting details away from the test, however I think this takes it a bit too far. The problem using `buildResponse` here is that it hides the details that the test cares about.

In this case, you are saying that the attributes match, however, one would have to go to `TestData` to see that. I would actually create both responses directly in this test.

Reply:
will fix that